test/libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -1702,6 +1702,72 @@ BOOST_AUTO_TEST_CASE(using_for)
 	checkCallGraphExpectations(get<1>(graphs), expectedDeployedEdges);
 }
 
+BOOST_AUTO_TEST_CASE(user_defined_binary_operator)
+{
+	unique_ptr<CompilerStack> compilerStack = parseAndAnalyzeContracts(R"(
+		type Int is int128;
+		using {add as +} for Int;
+
+		function add(Int, Int) returns (Int) {
+			return Int.wrap(0);
+		}
+
+		contract C {
+			function test() public {
```

Comment:
Going with the convention used in other tests here, I'd name the function in such a way to make its visibility obvious:
```suggestion
			function pub() public {
```

Same in the unary case.

Reply:
done